### PR TITLE
Add toggle between edit only/edit with preview/preview only

### PR DESCRIPTION
**Pull request by icewind1991**
Fixes https://github.com/icewind1991/files_markdown/issues/24

toggle

### CODE REVIEW

File: js/editor.js [hunk 1]
```diff
@@ -349,6 +350,54 @@ var Files_Texteditor = {
...
+			}
+			button.click(this.setPreviewMode.bind(this, type));
+			button.attr('data-type', type);
+			return button.css('background-image', 'url("' + OC.imagePath('files_texteditor', type) + '")');
+		}.bind(this);
+
+		var controls = $('<span/>').attr('id', 'preview_editor_controls');
+		controls.append(makeButton('text', t('files_texteditor', 'Edit')));
+		controls.append(makeButton('mixed', t('files_texteditor', 'Mixed'), true));
+		controls.append(makeButton('image', t('files_texteditor', 'Preview')));
```
Comment (MorrisJobke) [inline js/editor.js]: Those 3 images are not available.

Comment (icewind1991) [inline js/editor.js]: fixed

**Comment by MorrisJobke** [comment]
@icewind1991 sign-off missing 😉

**Comment by MorrisJobke** [comment]
Beside that 👍

**Pull request merged**